src/map/utils/battleutils.cpp

```diff
@@ -2020,7 +2020,7 @@ namespace battleutils
             if (giveTPtoVictim)
             {
                 //account for attacker's subtle blow which reduces the baseTP gain for the defender
-                float sBlowMult = ((100.0f - dsp_cap((float)PAttacker->getMod(Mod::SUBTLE_BLOW), 0.0f, 50.0f)) / 100.0f);
+                float sBlowMult = ((100.0f - (dsp_cap((float)PAttacker->getMod(Mod::SUBTLE_BLOW), 0.0f, 50.0f) + dsp_cap((float)PAttacker->getMod(Mod::SUBTLE_BLOW_II), 0.0f, 25.0f))) / 100.0f);
```

Comment:
Do you think it's a good assumption to think that it would work the same as PDT/MDT where MDT+MDT II caps at 87.5%?